AUTHORS

```diff
@@ -691,5 +691,14 @@ Xiaowei Li <446240525@qq.com>
 toastynerd <tylermorgan86@gmail.com>
 Todd Kennedy <toddself@users.noreply.github.com>
 Icer Liang <wizicer@users.noreply.github.com>
+Stephen Belanger <admin@stephenbelanger.com>
+Fishrock123 <fishrock123@rocketmail.com>
+Brendan Ashworth <brendan.ashworth@me.com>
+Andres Suarez <zertosh@gmail.com>
+Jonathan Ong <jonathanrichardong@gmail.com>
+ttrfwork <anton.kazmerchuk@gmail.com>
```

Comment:
@3674823674252 can you comment here on how you'd like your entry to appear in the AUTHORS file please?